### PR TITLE
es6ify the clientState module

### PR DESCRIPTION
**Pull request by pcardune**

### CODE REVIEW

File: apps/src/sites/studio/pages/code-studio.js [hunk 1]
```diff
@@ -32,7 +32,6 @@ require('@cdo/apps/code-studio/components/Attachments');
 $.fn.modal.Constructor.prototype.enforceFocus = function () {};
 
 window.dashboard = window.dashboard || {};
-window.dashboard.clientState = require('@cdo/apps/code-studio/clientState');
```
Comment (pcardune) [inline apps/src/sites/studio/pages/code-studio.js:35]: this is my favorite line!

File: apps/src/code-studio/clientState.js [hunk 2]
```diff
@@ -5,18 +5,17 @@
...
 var sessionStorage = window.sessionStorage;
 
-import { mergeActivityResult } from './activityUtils';
+import {mergeActivityResult} from './activityUtils';
+import {queryParams} from './utils';
 
-var clientState = module.exports = {};
-
-clientState.queryParams = require('./utils').queryParams;
+export {queryParams};
```
Comment (Bjvanminnen) [inline apps/src/code-studio/clientState.js:11]: ugh, we import a thing just so that we can export it?

**Comment by Bjvanminnen** [comment]
There are a few haml files where we call dashboard.clientState.reset(). Aren't those going to be broken now?

**Comment by pcardune** [comment]
drat, I always forget to search dashboard for these damn global calls. Thank you for pointing that out. Will fix.

**Comment by Bjvanminnen** [comment]
In most cases, that's exactly why these still live on the dashboard namespace. I made a push at one point to remove those that I could.

File: apps/src/sites/studio/pages/code-studio.js [hunk 3]
```diff
@@ -88,3 +87,8 @@ setTimeout(function () {
 activateReferenceAreaOnLoad();
 
 window.CDOSounds = new Sounds();
+
+import {trackProgress} from '@cdo/apps/code-studio/clientState';
+window.__TestHooks = {
```
Comment (Bjvanminnen) [inline apps/src/sites/studio/pages/code-studio.js]: elsewhere we've used `window.__TestInterface`

**Pull request closed without merging**